playbooks/roles/vhost/tasks/main.yml

```diff
@@ -59,7 +59,8 @@
     line: "ExecStop={{ COMMON_OBJECT_STORE_LOG_SYNC_ON_EXIT }}"
     insertafter: "^ExecStart=.*$"
   when: COMMON_OBJECT_STORE_LOG_SYNC
-        and not (ansible_distribution_release == 'precise' or ansible_distribution_release == 'trusty')
+        and ansible_distribution_release != 'precise'
```

Comment:
@Metfriet, I think it is better to leave the condition as it is since that is how it is present in the upstream PR and all the client-specific branches iirc.